tt_metal/impl/dispatch/kernels/cq_dispatch.cpp

```diff
@@ -162,7 +162,7 @@ void process_write_host_h() {
     // We will send the cmd back in the first X bytes, this makes the logic of reserving/pushing completion queue
     // pages much simpler since we are always sending writing full pages (except for last page)
     uint32_t length = cmd->write_linear_host.length;
-    DPRINT << "process_write_host_h: " << length << ENDL();
+    // DPRINT << "process_write_host_h: " << length << ENDL();
```

Comment:
Any reason why we're commenting out the dprints on cq_dispatch specifically?

Reply:
I was running into code size issues when running watcher/dprint fixtures